spec/helper.ts

```diff
@@ -0,0 +1,127 @@
+// The MIT License (MIT)
```

Comment:
nit: helper is a dirty word. Can we at least call it a `fooHelper`? Maybe `httpsBasedHelper`